pkg/cmd/admin/router/router.go

```diff
@@ -344,23 +345,24 @@ func generateSecretsConfig(cfg *RouterConfig, kClient *kclient.Client,
 			},
 		}
 		secrets = append(secrets, secret)
-		volume := kapi.Volume{
-			Name: "server-certificate",
-			VolumeSource: kapi.VolumeSource{
-				Secret: &kapi.SecretVolumeSource{
-					SecretName: secret.Name,
-				},
+	}
+
+	volume := kapi.Volume{
+		Name: "server-certificate",
+		VolumeSource: kapi.VolumeSource{
+			Secret: &kapi.SecretVolumeSource{
+				SecretName: fmt.Sprintf("%s-certs", cfg.Name),
```

Comment:
Compute the secret name in one place and use it in both the annotation value and the volume definition